### PR TITLE
fix(switch): update selected background color tokens

### PR DESCRIPTION
**Pull request by jenndiaz**
## Description
Updated selected (not emphasized) background color tokens to use: 
- neutral-background-color-selected-default
- neutral-background-color-selected-hover
- neutral-background-color-selected-down
- neutral-background-color-selected-key-focus

Updated storybook to include disabled and emphasized variants and controls for sizing and label

## How and where has this been tested?

Please tag yourself on the tests you've marked complete to confirm the tests have been run by someone other than the author.

#### Test outline:
  1. Open the [storybook](https://pr-2057--spectrum-css.netlify.app/preview/?path=/docs/components-switch--docs) for the Switch component:
- [x] All variants load without error and displayed appropriately @mlogsdon18 
  
2. Open the [Docs Site](https://pr-2057--spectrum-css.netlify.app/switch.html) for the Switch component:
- [x] Inspect and confirm new tokens are being used for the selected not emphasized variants @mlogsdon18 

### Regression testing

Validate:

1. A legacy documentation page (such as [accordion](https://pr-2057--spectrum-css.netlify.app/accordion.html)), including:

- [x] The page renders correctly
- [x] The page is accessible
- [x] The page is responsive
@mlogsdon18 
2. A migrated documentation page (such as [action group](https://pr-2057--spectrum-css.netlify.app/actiongroup.html)), including:

- [x] The page renders correctly
- [x] The page is accessible
- [x] The page is responsive
@mlogsdon18 

## To-do list

- [x] I have read the [contribution guidelines](/.github/CONTRIBUTING.md).
- [x] I have updated relevant storybook stories and templates.
- [x] I have tested these changes in Windows High Contrast mode.

- [x] If my change impacts **other components**, I have tested to make sure they don't break.
- [x] If my change impacts **documentation**, I have updated the documentation accordingly.

- [ ] ✨ This pull request is ready to merge. ✨

### CODE REVIEW

**Review by mlogsdon18 — COMMENTED** [review]
Looks great! Just one typo with storybook but otherwise looks good!

File: components/switch/stories/switch.stories.js [hunk 1]
```diff
@@ -6,9 +6,57 @@ export default {
...
+			table: {
+				type: { summary: "string" },
+				category: "Content",
+			},
+			control: { type: "text" },
+		},
+	},
 	args: {
 		rootClass: "spectrum-Switch",
+		isDisabled: false,
```
Comment (jawinn) [inline components/switch/stories/switch.stories.js:59]:
```suggestion
		isDisabled: false,
		isChecked: false,
```

File: components/switch/stories/switch.stories.js [hunk 2]
```diff
@@ -24,3 +72,13 @@ export default {
...
+
+export const Emphasized = Template.bind({});
+Emphasized.args = {
+	isEmphasized: true
+};
+
+export const Disabled = Template.bind({});
+Disabled.args = {
+	isDisabled: true
+};
```
Comment (jawinn) [inline components/switch/stories/switch.stories.js:84]: Could you add a "Checked" story as well?

File: components/switch/stories/template.js [hunk 3]
```diff
@@ -7,9 +7,11 @@ import "../index.css";
 export const Template = ({
 	rootClass = "spectrum-Switch",
 	size = "m",
+	label = "Switch label",
+	isDisabled,
```
Comment (jawinn) [inline components/switch/stories/template.js:11]:
```suggestion
	isDisabled,
	isChecked,
```

File: components/switch/stories/template.js [hunk 4]
```diff
@@ -25,13 +27,15 @@ export const Template = ({
...
+				[`${rootClass}--disabled`] : isDisabled,
+				[`${rootClass}--emphasized`] : isEmphasized,
 				[`${rootClass}--size${size?.toUpperCase()}`]:
 					typeof size !== "undefined",
 				...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
 			})}
 			id=${ifDefined(id)}
 		>
-			<input type="checkbox" class="${rootClass}-input" id="switch-onoff-0" />
+			<input type="checkbox" class="${rootClass}-input" id="switch-onoff-0" ?disabled=${isDisabled} />
```
Comment (jawinn) [inline components/switch/stories/template.js:38]:
```suggestion
			<input type="checkbox" class="${rootClass}-input" id="switch-onoff-0" ?disabled=${isDisabled} ?checked=${isChecked} />
```

**Review by jawinn — COMMENTED** [review]
Looks good except for what Megan noted. Could you also incorporate `isChecked` into the template and add a new "Checked" story?

File: components/switch/stories/switch.stories.js [hunk 5]
```diff
@@ -6,9 +6,57 @@ export default {
...
+			name: "Disabled",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "State",
+			},
+			control: "boolean",
+		},
+		isChecked: {
+			name: "Disabled",
```
[the remark this answers is not in the document]
Comment (jenndiaz) [reply components/switch/stories/switch.stories.js]: oh good catch, fixed!

File: components/switch/stories/template.js (hunk 4 above)
Comment (jenndiaz) [reply components/switch/stories/template.js]: yes, good call, I've added a `Checked` story

**Review by jawinn — APPROVED** [review]
Looks great. Thanks for adding that!

**Pull request merged**